test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,79 @@
+./test_scripts/API/SetAudioStreamingIndicator/043_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_RPC_omitted_PTgroups.lua Check_that_SDL_responds_with_DISALLOWED_when_RPC_omitted_PT_groups_navi_app
+./test_scripts/API/SetAudioStreamingIndicator/044_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_RPC_omitted_PTgroups.lua Check_that_SDL_responds_with_DISALLOWED_when_RPC_omitted_PT_groups_comm_app
+./test_scripts/API/SetAudioStreamingIndicator/045_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_FULL_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/046_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_LIMITED_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/047_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_BACKGROUND_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/048_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_NONE_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/049_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_FULL_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_non-media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/050_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_BACKGROUND_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_non-media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/051_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_NONE_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_non-media_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/052_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_FULL_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_navi_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/053_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_LIMITED_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_navi_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/054_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_BACKGROUND_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_navi_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/055_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_NONE_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_navi_notincluded_PT
```

Comment:
@istoimenova If your idea is check app in way: 
[NONE, BACKGROUND, LIMITED] -> check in FULL
[NONE, BACKGROUND, FULL] -> check in LIMITED
[NONE, LIMITED, FULL] -> check in BACKGROUND
[BACKGROUND, LIMITED, FULL] -> check in NONE it make sense